src/shared/components/event-meta/index.js

```diff
@@ -9,8 +9,11 @@ const EventMeta = ({
   externalLinks,
   tags,
 }) => {
+  let linksSection = (<noscript />);
+  let tagsSection = (<noscript />);
+
   if (internalLinks.length > 0 && externalLinks.length > 0) {
```

Comment:
This looks like an array type check to me. If so use `Array.isArray()`. I would rather use default values over type checks though.